hip/static/styles/init.scss

```diff
@@ -0,0 +1,12 @@
+/* Initialization File
+
+This file will manage all scss file imports.
+*/
+
+// ------- bulma.io css framework ----------------- //
+
+@import "https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css";
```

Comment:
Minor note: it looks like version 0.9.2 has been released

Reply:
will update

Reply:
done